src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -175,11 +177,16 @@ void Terminal::UpdateAppearance(const ICoreAppearance& appearance)
     _defaultBg = newBackgroundColor.with_alpha(0);
     _defaultFg = appearance.DefaultForeground();
     _intenseIsBright = appearance.IntenseIsBright();
+    _perceptualColorNudging = appearance.PerceptualColorNudging();
 
     for (int i = 0; i < 16; i++)
     {
         _colorTable.at(i) = til::color{ appearance.GetColorTableEntry(i) };
     }
+    if (_perceptualColorNudging)
+    {
+        _MakeAdjustedColorArray();
+    }

```

Comment:
might need to do an 
`else 
{
    _adjustedColorArray.clear()
}`
to ensure it doesn't persist across settings reloads